packages/components/Avatar/src/useAvatar.ts

```diff
@@ -126,3 +140,18 @@ export const removeRedundantCharacters = (name: string): string[] => {
 export const removeTitlesFromName = (words: string[]): string[] => {
   return words.filter((word) => !titles.has(word));
 };
+
+function getIconSize(size: AvatarSize) {
```

Comment:
Can this be named "getFontIconSize" to make it clear it's specific to font icons?

Reply:
fixed